faiss-search/scann/scann.py

```diff
@@ -83,12 +83,12 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
             tree(self.num_leaves, self.num_leaves_to_search, self.training_sample_size).\
             score_ah(self.dimensions_per_block, self.anisotropic_quantization_threshold).\
             reorder(self.reordering_num_neighbors).create_pybind()
-        return self.searcher
+        return searcher
 
     def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
         if self.reordering_num_neighbors < self.top_k:
```

Comment:
This condition should be reversed

Reply:
I thought `reordering_num_neighbors` be higher than k? They have this on their documentation:
"`reordering_num_neighbors` should be greater than k, the final number of neighbors."

Reply:
yes sorry, my bad. Ignore this